contracts/core/space.cairo

```diff
@@ -57,6 +50,18 @@ end
 func ship(id : felt) -> (contract_address : felt):
 end
 
+@storage_var
+func dust_generation_seed() -> (count : felt):
```

Comment:
I don't get was this storage var holds – I don't see the link between `dust_generation_seed` and `count`